# Header
Header header
# Is it complete (0) or partial update (!=0)
uint8 isPartial
# Camera information
sensor_msgs/CameraInfo camera_info
# Pointcloud
sensor_msgs/PointCloud2 pointcloud2